tests/test_ge_utils.py

```diff
@@ -557,3 +559,21 @@ def test_convert_ndarray_float_to_datetime_tuple(
 def test_hyphen():
     txt: str = "validation_result"
     assert hyphen(txt=txt) == "validation-result"
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "data", [pytest.param({"t": datetime.time(hour=1, minute=30, second=45)}, id="datetime.time")]
+)
+def test_convert_to_json_serializable_converts_correctly(data: dict):
+    ret = convert_to_json_serializable(data)
+    assert ret == {"t": "01:30:45"}
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "data", [pytest.param({"t": datetime.time(hour=1, minute=30, second=45)}, id="datetime.time")]
+)
+def test_ensure_json_serializable(data: dict):
+    ensure_json_serializable(data)
+    # Passes if no exception raised
```

Comment:
tests look good, but they should probably live in `tests.test_convert_to_json_serializable`, unless there's a compelling reason to break the pattern.

Reply:
Thanks, I'll move. I didn't notice that file assuming we were following the pattern of test file per module

Reply:
yeah, that would be super reasonable - at some point we could refactor this to be more logical. thanks for addressing!